### PR TITLE
changed service messages references

### PR DESCRIPTION
**Pull request by OgunyemiO**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/EXUI-1079

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by RiteshHMCTS — DISMISSED** [review]

**Review by Josh-HMCTS — DISMISSED** [review]

**Review by andywilkinshmcts — APPROVED** [review]

**Comment by andywilkinshmcts** [comment]
Needs updating to point to a final version of common lib, but ready for QA

**Review by Josh-HMCTS — DISMISSED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by andywilkinshmcts — APPROVED** [review]
Make sure that a full release of common lib is done before this is merged

**Review by Josh-HMCTS — APPROVED** [review]

**Review by Josh-HMCTS — APPROVED** [review]

**Review by RiteshHMCTS — DISMISSED** [review]

**Review by RiteshHMCTS — DISMISSED** [review]

**Review by anthonydummer — APPROVED** [review]
LGTM

File: .yarnrc.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+enableStrictSsl: false
```
Comment (RiteshHMCTS) [inline .yarnrc.yml:1]: you need to remove this line

**Review by StoneBenHMCTS — DISMISSED** [review]

**Review by RiteshHMCTS — DISMISSED** [review]

**Review by olusegz07 — DISMISSED** [review]

**Review by andywilkinshmcts — APPROVED** [review]
Make sure that you release a new common lib full version and update this PR before this PR is merged

**Review by Josh-HMCTS — APPROVED** [review]

**Review by RiteshHMCTS — APPROVED** [review]

**Pull request merged**